src/background/deinflect.ts

```diff
@@ -951,7 +124,294 @@ interface DeinflectRule {
   fromType: number;
   toType: number;
   reason: Reason;
-}
+};
+
+// prettier-ignore
+const deinflectRuleData: Array<DeinflectRule> = [
```

Comment:
Sorry! I meant that we should just change the typing here to be a named tuple, not that we should change it to use `DeinflectRule`.

i.e.

```diff
-const deinflectRuleData: Array<[string, string, number, number, Reason]> = [
+const deinflectRuleData: Array<
+  [from: string, to: string, fromType: number, toType: number, reason: Reason]
+> = [
```

Reply:
Ah, my bad. This change makes a lot more sense to me now :)